kas/iot2050.yml

```diff
@@ -27,7 +27,7 @@ repos:
 
   isar:
     url: https://github.com/ilbers/isar
-    refspec: f5a6fdfd9aad49202093aab2158f625429eaf7c5
+    refspec: 03124cca669f50b682336a0bdf4ede5a4238e144
```

Comment:
I have an update pending that includes https://groups.google.com/d/msgid/isar-users/185f97a0-6755-1344-43cd-7df51040ba65%40siemens.com so that we do not get the wic warning "WARNING: iot2050-image-example-1.0-r0 do_wic_image: WARNING: cannot expand variable board_name". Just pushed that wip branch https://github.com/siemens/meta-iot2050/commits/jan/container-sdk.

Reply:
Thanks will look into it.